learning_journal/learning_journal/routes.py

```diff
@@ -1,7 +1,6 @@
 def includeme(config):
-    """ This function adds routes to Pyramid's Configurator """
-    config.add_static_view('static', 'static', cache_max_age=3600)
+    """ This function adds routes to Pyramid's Configurator. """
     config.add_route('home', '/')
     config.add_route('detail', '/{id:\d+}')
```

Comment:
Not gonna ding you on this, but having your "detail" route this simple is a bit of an issue and here's why: from the outside looking in I can't tell what I'm supposed to be accessing. Just like with variables, function names, class names, HTML, css selectors, etc., routes should be semantic, describing the thing that they access.